app/controllers/catalog_controller.rb

```diff
@@ -131,9 +131,10 @@ def guest_uid_authentication_key(key)
     # or can be specified manually to be different.
 
     keyword_fields = [
-      'isbn_ssim', 'issn_ssim', 'lccn_ssim', 'id', 'oclc_ssim', 'other_standard_ids_ssim',
-      'publisher_number_ssim', 'nonformat_table_contents_tesim', 'summary_tesim',
-      'participant_performer_note_tesim', 'creation_production_credits_tesim', 'local_note_tesim'
+      'isbn_ssim', 'id', 'title_display_tesim', 'title_vern_display_tesim', 'title_addl_tesim', 'title_added_entry_tesim',
+      'title_series_tesim', 'subtitle_display', 'subtitle_vern_display', 'author_display_tesim', 'author_vern_display_tesim',
```

Comment:
@alexBLR Can you update `subtitle_display` and `subtitle_vern_display` to `_tesim`/`_tesi` in blacklight-catalog/lib/tasks/solrmarc/index.properties and remove them in the blacklight-catalog/solr/conf/schema.xml so that we're dynamically defining them in the schema? Right now, there implicitly defined as strings, which won't help us for tokenized searching.

Reply:
👍